shared/models/HOKS.ts

```diff
@@ -243,7 +243,10 @@ export const HOKS = types
             ),
             credentials: "include",
             body: JSON.stringify({
-              "oppilaitos-oid": self.opiskeluOikeus.oppilaitos.oid
+              "oppilaitos-oid":
+                self.opiskeluOikeus?.oppilaitos?.oid === ""
+                  ? null
+                  : self.opiskeluOikeus?.oppilaitos?.oid
```

Comment:
Tyhjä merkkijono on muuten falsy, ja siksi tämän voi myös kirjoittaa:

```suggestion
                self.opiskeluOikeus?.oppilaitos?.oid || null
```